pokesets/Manga/Electric Tale of Pikachu/ET-Other.yaml

```diff
@@ -0,0 +1,42 @@
+species: Haunter
+setname: ET-Other
+ingamename: Black Fog
+displayname: Black Fog
+ability: [null]
+item: [null]
+ball: [Ultra]
+moves:
+- [Dream Eater]
```

Comment:
Weird that it would use Dream Eater in the manga and not, you know, have any sleep moves. Would still work with doubles and Rest tho